CHANGELOG.md

```diff
@@ -5,14 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
-## [7.3.1 - 2021-05-17]
-### Fixed
-- Bug #1: Verbindung zu dem/den Suchbroker(n) prüfen: Testanfrage herunterladen und ausführen
-- Bug #2: Verbindung zu dem/den Suchbroker(n) prüfen: Erreichbarkeit prüfen
-- Bug #3: Verbindung zum ID-Manager prüfen: Export ID anfordern
-- Code Style Fix
 
-## [7.3.0 - 2021-03-04]
```

Comment:
Don't change the date of the released versions. Please add a new date

Reply:
David told me to do it this way. Now i am confused...